Dockerfile

```diff
@@ -45,17 +48,28 @@ RUN set -eux ; \
         file \
         curl-dev \
         libpng-dev \
+        jpeg-dev \ 
+        freetype-dev \
+	libwebp-dev \
+	libzip-dev \
         imagemagick-dev; \
     docker-php-ext-install -j$(nproc) \
     pdo  \
     pdo_pgsql \
     intl \
     pcntl  \
     curl \
+    gd \
     exif ; \
     pecl install amqp ; \
     pecl install xdebug; \
     pecl install imagick ; \
+    docker-php-ext-configure gd   \
```

Comment:
this must be before docker-php-ext-install

Reply:
Moved before docker-php-ext-install